libr/util/str.c

```diff
@@ -216,7 +216,7 @@ R_API int r_str_binstr2bin(const char *str, ut8 *out, int outlen) {
 // Returns the permissions as in integer given an input in the form of rwx, rx,
 // etc.
 R_API int r_str_rwx(const char *str) {
-	int ret = atoi (str);
```

Comment:
looks to me this is not needed because you should always pass a "rwx" string, not directly a number, am i wrong?